recipes/libx264/all/conanfile.py

```diff
@@ -12,7 +12,7 @@ class LibX264Conan(ConanFile):
     license = "GPL-2.0"
     settings = "os", "arch", "compiler", "build_type"
     options = {"shared": [True, False], "fPIC": [True, False], "bit_depth": [8, 10, "all"]}
```

Comment:
You should also add the option to `options`.
```suggestion
    options = {"shared": [True, False], "fPIC": [True, False], "bit_depth": [8, 10, "all"], 'disable_asm': [True, False]}
```